server/conn.go

```diff
@@ -603,6 +614,7 @@ func (cc *clientConn) handleQuery(sql string) (err error) {
 	} else {
 		loadDataInfo := cc.ctx.Value(executor.LoadDataVarKey)
 		if loadDataInfo != nil {
+			defer cc.ctx.SetValue(executor.LoadDataVarKey, nil)
 			if err = cc.handleLoadData(loadDataInfo.(*executor.LoadDataInfo)); err != nil {
 				return errors.Trace(err)
```

Comment:
I think it's OK to use `break`. And I want to put this logic in the same function.